manifests/cfcr.yml

```diff
@@ -1,43 +1,59 @@
-name: ((deployment_name))
-
-features:
-  use_dns_addresses: true
+name: cfcr
 
 releases:
+- name: kubo
+  version: 0.9.0
+  url: https://github.com/cloudfoundry-incubator/kubo-release/releases/download/v0.9.0/kubo-release-0.9.0.tgz
+  sha1: e88722e12cce75b7db688da42564fc3f61715dbf
 - name: kubo-etcd
   version: 4
   url: https://github.com/pivotal-cf-experimental/kubo-etcd/releases/download/v4/kubo-etcd.4.tgz
   sha1: f94ab00d423765e2c740836788dc256caa1af014
-- name: kubo
-  version: latest
 - name: docker
   version: 30.1.4
   url: https://github.com/cloudfoundry-community/docker-boshrelease/releases/download/v30.1.4/docker-30.1.4.tgz
   sha1: f79a70452b3ddae52f5ac78dd37ff8d9e6ec19e2
-- name: haproxy
-  version: 8.4.0
-  url: https://github.com/cloudfoundry-incubator/haproxy-boshrelease/releases/download/v8.4.0/haproxy-8.4.0.tgz
-  sha1: a3a911f0cf8e672b27c6cb16318fd8c7c77f5bde
+- name: bosh-dns
+  sha1: e38d3e5edd16ce2dca778440b636ab1ea61f892f
+  url: https://bosh.io/d/github.com/cloudfoundry/bosh-dns-release?v=0.0.11
+  version: 0.0.11
 
 stemcells:
 - alias: trusty
   os: ubuntu-trusty
-  version: "3468.5"
+  version: 3468.latest
```

Comment:
The CI pipeline is failing due to the following error:
`Error: No stemcell found at 'https://s3.amazonaws.com/bosh-core-stemcells/google/bosh-stemcell-3468.latest-google-kvm-ubuntu-trusty-go_agent.tgz'.`